app/controllers/courses_controller.rb

```diff
@@ -116,4 +126,12 @@ def assign_show_view_vars
   def set_organization
     @organization = Organization.find_by(slug: params[:organization_id])
   end
+
+  def disable_course(disabled)
+    @course = Course.find(params[:id])
+    @course.disabled = disabled
+    notice_action = disabled ? 'disabled' : 'enabled'
+    notice_message = (@course.save ? 'Course was successfully' : 'Course could not be') + " #{notice_action}."
+    redirect_to(organization_course_path(@organization, @course), notice: notice_message)
```

Comment:
Is there a need to have the the course as an instance variable, isn't a local variable good enough